transcom/mymove
# MB-16088: Make estimated weight actually optional shuttle service items during payment requests

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16088)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1) Find a new move as a TOO and approve it. You may need to fill in any missing orders information.
2) Add a shuttle service item as the prime, either in the prime simulator or using postman. It should not matter if it is origin or destination. **Include the actual weight, but not the estimated weight.**
3) Go back as the TOO and approve the shuttle service item under the "Move task order" tab by clicking the "Accept" button.
4) Go back as the prime and try to create a payment request for that shuttle service item. It should be successfully created.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
